docs/Features.md

```diff
@@ -11,7 +11,7 @@ React-admin is a **rich framework** that covers most of the needs of typical adm
 
 With react-admin, developers assemble application components without having to worry about low-level details. They need less code for the same result, and they can **focus on the business logic** of their app.
 
-[![List view without and with react-admin](./img/list-from-react-to-react-admin.webp)](./ListTutorial.md#from-pure-react-to-react-admin)
```

Comment:
Not sure about this, shouldn't this be an image with a link?

Reply:
The current behaviour of this image link is to go to a 404 page. This is the current url: .../react-admin/ListTutorial.md#from-pure-react-to-react-admin. It points to a md file instead of a html one. SO the quickest fix was to link to the image.

Reply:
Besides, with the previous version it took 2 clicks to open the image fullscreen, whereas now it only takes one, which provides a better UX IMO

Reply:
Then we should remove the link altogether no?
```
![List view without and with react-admin](./img/list-from-react-to-react-admin.webp)
```

Reply:
I do not know. My first feeling was to click the image in order to see better the source codes.

Reply:
It seems we do not use this behaviour elsewhere, but I think it's a good idea to provide a link to open images in fullscreen

Reply:
The link to the full-size image is better, so let's keep this modification.